src/components/code-editor/code-editor.scss

```diff
@@ -227,4 +229,21 @@
         }
         border-radius: 50%;
     }
+
+    .CodeMirror-foldmarker {
```

Comment:
![Screen Shot 2022-08-31 at 15 56 08](https://user-images.githubusercontent.com/435885/187696277-5c0493ff-e744-40f3-ba72-a715c2f5e129.png)

This is barely visible on my machine. (I only get the dark mode)

Also, can we use something else for the arrows in the gutter instead of the default (ugly) ones?
I found that they are given by this style:

```css
.CodeMirror-foldgutter-open::after {
  content: "▾";
}
```